package.json

```diff
@@ -55,13 +57,24 @@
     "serve": "gatsby serve",
     "deployStaging": "gatsby build && echo 'staging.openousd.org' > public/CNAME && gh-pages -d public",
     "promoteStaging": "./scripts/promote_staging.sh",
-    "refreshData": "python scripts/build_json.py"
+    "refreshData": "python scripts/build_json.py",
+    "percyStaging": "PERCY_TOKEN=$(grep PERCY_TOKEN .env.production | cut -d '=' -f2) PERCY_BRANCH=main percy snapshot --base-url=https://staging.openousd.org  scripts/testing/snapshots.js",
+    "percyProd": "PERCY_TOKEN=$(grep PERCY_TOKEN .env.production | cut -d '=' -f2) PERCY_BRANCH=main percy snapshot --base-url=https://openousd.org scripts/testing/snapshots.js"
```

Comment:
The grep is a hack to get the PERCY API token from a config file instead of committing it to source here. It won't work on Windows.

Only other difference is setting staging or prod base url. Not sure if there is a way to pass that as a flag through npm to percy CLI.